client/blocks/product-selector/index.jsx

```diff
@@ -222,6 +248,20 @@ export class ProductSelector extends Component {
 		);
 	}
 
+	getRelatedMonthlyProductSlug( yearlyProductSlug ) {
+		const { productPriceMatrix } = this.props;
+
+		if ( ! productPriceMatrix ) {
+			return null;
+		}
+
+		if ( ! productPriceMatrix[ yearlyProductSlug ] ) {
```

Comment:
Why not combine those two related checks into a single one like:
```js
if ( ! productPriceMatrix || ! productPriceMatrix[ yearlyProductSlug ] ) {
	return null;
}
```

Reply:
Good call 👍 Done in https://github.com/Automattic/wp-calypso/pull/37396/commits/a123b5f15c4a6fd0b5321dd995f3328b388678a8.

Reply:
Good call 👍 Done in a123b5f.